membrane_curvature/lib/mods.py

```diff
@@ -1,150 +1,44 @@
-        Leaflets of bilayer.
-    head_list : list
-        List of indexes by leaflet. Provided by user in -ii and -io.
-    topology: obj
-        Topology from grofile.
-    Returns
-    -------
-    [ element_1, ..., element_n] : list
-        List with elements used as a reference to derive membrane surface.
+    x: float
+        Value of x coordinate
+    y: float
+        Value of y coordinate
 
-    """
+        Returns
+        -------
+        Return a tuple [l,m] with l,m as int.
 
-    dic_all_beads = {lf: {lt: [] for lt in lipid_types} for lf in leaflets}
-    print('==== Lipid types in membrane ==== ')
-    for lt in lipid_types:
-        print('====>', lt)
-        dic_all_beads['upper'][lt] = np.concatenate((topology.select('resname ' +
-                                                                     lt +
-                                                                     ' and index ' +
-                                                                     str(head_list[0]) +
-                                                                     ' to ' +
-                                                                     str(head_list[1]) +
-                                                                     ' and name PO4'), topology.select('resname ' +
-                                                                                                       lt +
-                                                                                                       ' and index ' +
-                                                                                                       str(head_list[0]) +
-                                                                                                       ' to ' +
-                                                                                                       str(head_list[1]) +
-                                                                                                       ' and name GM1'))).astype(int).tolist()
-        dic_all_beads['lower'][lt] = np.concatenate((topology.select(
-            'resname ' + lt + ' and index ' + str(head_list[1] + 1) + ' to ' + str(head_list[2]) + ' and name PO4'),
-            topology.select(
-            'resname ' + lt + ' and index ' + str(head_list[1] + 1) + ' to ' + str(head_list[2]) + ' and name GM1'))).astype(int).tolist()
-
-        print("upper", len(dic_all_beads['upper'][lt]))
-        print("lower", len(dic_all_beads['lower'][lt]))
-
-    return dic_all_beads
-
-
-def dict2pickle(name, dict_):
     """
-    Exports values stored in a dictionary to a pickle file.
-
-    Parameters
-    ----------
-    name : str,
-        name of pickle file.
-    dict_ : dict like {index -> value}
-        dictionary to export.
 
-    Returns
-    -------
-        Returns a pickled dictionary.
+    index_grid_l = int(abs(coords[0]) * factor)
+    index_grid_m = int(abs(coords[1]) * factor)
```

Comment:
Is coords a single coordinate (essentially a 3-tuple)?

If so then you should keep in mind for later that you can almost certainly gain quite a bit of performance by doing the grid indexing operation on all coordinates at once (there's code related to `np.histogramdd` that you can probably use... but not in this PR!)